README.md

```diff
@@ -65,12 +65,36 @@ make run
     make deploy
     ```
 
+### Access the multicluster-controlplane
+
 After the multicluster-controlplane is deployed, getting the multicluster-controlplane kubeconfig with following command to access the multicluster-controlplane
 
 ```bash
 kubectl -n multicluster-controlplane get secrets multicluster-controlplane-kubeconfig -ojsonpath='{.data.kubeconfig}' | base64 -d > multicluster-controlplane.kubeconfig
 ```
 
+Or if you deployed the multicluster-controlplane on an Openshift Cluster, you can delegate the authentication with your cluster kube-apiserver, run following command to set a context in your cluster kubeconfig
+
+1. Login to your cluster with `oc login`
+
+2. Create a new context in your cluster kubeconfig
+
+    ```bash
+    route=$(oc -n multicluster-controlplane get route multicluster-controlplane -o jsonpath='{.status.ingress[0].host}')
+
+    # if you want to add the ca of your cluster kube-apiserver, using the command:
+    # oc config set-cluster multicluster-controlplane --server="https://${route}" --embed-certs --certificate-authority=<the ca path of your cluster kube-apiserver>
```

Comment:
can we provide a command to get the openshift ca? if so, we can remove `--insecure-skip-tls-verify`.

Reply:
yes, we can, will update this part

Reply:
updated